frontend/app/routes/$lang+/_public+/apply+/$id+/date-of-birth.tsx

```diff
@@ -45,7 +52,7 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   const csrfToken = String(session.get('csrfToken'));
   const meta = { title: t('gcweb:meta.title.template', { title: t('apply:eligibility.date-of-birth.page-title') }) };
 
-  return json({ id: state.id, csrfToken, meta, defaultState: state.dateOfBirth, editMode: state.editMode });
+  return json({ id: state.id, csrfToken, meta, defaultState: state, editMode: state.editMode });
```

Comment:
Only return what's relevant to this page for `defaultState`.

```suggestion
 const {dateOfBirth, allChildrenUnder18} = state;
  return json({ id: state.id, csrfToken, meta, defaultState: {dateOfBirth, allChildrenUnder18}, editMode: state.editMode });
```